lib/template/code_example_templates/js_example_template.html.erb

```diff
@@ -0,0 +1,7 @@
+<script><%= rendered_example %></script>
+<div class="codeBlock jsExample">
+  <div class="highlight">
+    <pre><%= code_example %></pre>
+  </div>
+</div>
+
```

Comment:
Yes. They might just have to create a factory file, if they can use one of the default included templates ('markup_example_template', 'js_example_template')